sdk/servicebus/azure-servicebus/tests/async_tests/test_queues_async.py

```diff
@@ -81,6 +81,7 @@ async def test_async_queue_by_queue_client_send_multiple_messages(self, serviceb
                     message = Message("Handler message no. {}".format(i))
                     messages.append(message)
                 await sender.send_messages(messages)
+                assert sender._handler._msg_timeout == 0
```

Comment:
Did we define what a timeout of 0 means? If so - we should update the docstring to make it clear.

Reply:
added the docstring that the timeout value must be greater than 0, and if timeout is 0, we would raise error now.